### DISCUSSION TITLE
`lines-between-class-members` and `exceptBetweenSingleLine`

### DISCUSSION THREAD

**Discussion by tukusejssirs** [eslint/eslint#16142] [Q&A] [answered]
I want to request a feature to only enforce newlines between two multiline class members and a multiline class member and a signleline class member, while prohibiting newlines between two singleline class members.

However, I’ve found #14665 which dealt with exactly the same request. The [final comment](https://github.com/eslint/eslint/issues/14665#issuecomment-855155206) from @nzakas that _[p]er [y]our [stylistic rule policy](https://github.com/eslint/eslint#stylistic-rule-updates), [you] aren’t adding any more options to this rule_.

I understand that a stylistic rule policy is something required to have and you need to keep it as thoroughly as possible, however, I want to know if it is possible to somehow configure `eslint` along the lines of my first paragraph of this comment. Is there any other `eslint` rule I could use? Or a plugin?

Thanks in advance! :pray:

**Comment by nzakas** [comment, marked as answer]
Your best bet is to write your own rule for that.
